components/02-elements/tab/_tab.scss

```diff
@@ -1,99 +1,130 @@
-$tab__padding                   : 15px !default;
-$tab__margin                    : 10px !default;
-$tab__title-border              : 1px solid $gray-light !default;
-$tab__title-background          : $gray-lightest !default;
-$tab__title-border-color--active: $gray-light !default;
-$tab__title-border-width--active: 1px 1px 0 1px !default;
-$tab__title-border-style--active: solid !default;
-$tab__title-background--active  : $white !default;
-$tab__icon-width                : 13px !default;
-$tab__icon-height               : 13px !default;
-$tab__icon-fill                 : $gray !default;
-$tab__icon-transition           : transform 0.3s !default;
-$tab__content-transition        : opacity 0.5s !default;
-
+$tab__border-width                      : 8px !default;
+$tab__border-width--mobile              : 1px !default;
+// title
+$tab__title-padding                     : $spacer--medium !default;
+$tab__title-border                      : $tab__border-width solid $gray-light !default;
+$tab__title-border--active              : $tab__border-width solid $color-primary !default;
+$tab__title-border--mobile              : $tab__border-width--mobile solid $gray-light !default;
+$tab__title-color                       : $color-secondary !default;
+$tab__title-color-mobile-active         : $color-primary !default;
+$tab__title-font-weight                 : $font-weight-normal !default;
+$tab__title-font-weight--mobile         : $font-weight-bold !default;
+$tab__title-font-weight--active         : $font-weight-bold !default;
+$tab__title-background                  : $bg-color-base !default;
+$tab__title-background--active          : $gray-light !default;
+$tab__title-background--mobile-active   : $color-secondary !default;
+// content
+$tab__content-padding                   : $spacer--large $spacer--extra-large !default;
+$tab__content-padding--mobile           : $spacer--medium $spacer--large !default;
```

Comment:
It's mobile first, so you should add a modificator, to the larger screen and you shouldn't use `mobile` / `desktop` / `phone` / `tablet` and other words that describe device itself. Use `small` / `medium` / `large` instaed.

Reply:
Done